editor/src/components/canvas/canvas-strategies/strategies/ancestor-metastrategy.tsx

```diff
@@ -0,0 +1,119 @@
+import { MetadataUtils } from '../../../../core/model/element-metadata-utils'
+import * as EP from '../../../../core/shared/element-path'
+import { CSSCursor } from '../../canvas-types'
+import { highlightElementsCommand } from '../../commands/highlight-element-command'
+import { setCursorCommand } from '../../commands/set-cursor-command'
+import { MetaCanvasStrategy } from '../canvas-strategies'
+import {
+  getTargetPathsFromInteractionTarget,
+  InteractionCanvasState,
+  targetPaths,
+} from '../canvas-strategy-types'
+
+export function ancestorMetaStrategy(
+  allOtherStrategies: Array<MetaCanvasStrategy>,
+  level: number,
+): MetaCanvasStrategy {
+  return (canvasState, interactionSession, customStrategyState) => {
+    // Don't apply during insertion
+    if (canvasState.interactionTarget.type === 'INSERTION_SUBJECTS') {
+      return []
+    }
+
+    const targets = getTargetPathsFromInteractionTarget(canvasState.interactionTarget)
+
+    if (targets.length !== 1) {
+      return []
+    }
+
+    const target = targets[0]
+
+    // Avoid children of the storyboard
+    if (
+      EP.isEmptyPath(targets[0]) ||
```

Comment:
```suggestion
      EP.isEmptyPath(target) ||
```

Reply:
Ha! I think this was the result of some code being shuffled around